src/gui/FileBrowser.cpp

```diff
@@ -403,11 +340,11 @@ void FileBrowserTreeWidget::contextMenuEvent(QContextMenuEvent * e )
 						this,
 					SLOT( sendToActiveInstrumentTrack() ) );
 		contextMenu.addAction( tr( "Open in new instrument-track/"
-								"Song-Editor" ),
+								"Song Editor" ),
 						this,
 					SLOT( openInNewInstrumentTrackSE() ) );
 		contextMenu.addAction( tr( "Open in new instrument-track/"
-								"B+B Editor" ),
+								"Pattern Editor" ),
```

Comment:
@tresf Oh. That's awkward. I saw `View -> Pattern Editor` had been updated, so I assumed these were leftover areas that had been missed. But now I see many/most places still use B&B editor (e.g. right-clicking on a bassline pattern, and also the default B&B pattern names & the window title for the B&B editor).

Was the change in the view menu a rogue edit? I can submit rollback patches for that plus the change I made, if desired.